grafana-plugin/src/pages/users/Users.tsx

```diff
@@ -74,18 +75,20 @@ class Users extends React.Component<UsersProps, UsersState> {
     const { usersFilters, page } = this.state;
     const { userStore } = store;
 
-    if (!isUserActionAllowed(UserActions.UserSettingsWrite)) {
+    if (!isUserActionAllowed(REQUIRED_PERMISSION_TO_VIEW_USERS)) {
+      this.setState({ initialUsersLoaded: true });
```

Comment:
actually we don't load users if `!isUserActionAllowed(REQUIRED_PERMISSION_TO_VIEW_USERS)`, why do we need set `initialUsersLoaded` to`true`? looks contradictory

Reply:
`initialUsersLoaded` is used to conditionally show the `emptyText` on the `GTable` component.

But you're right, the fact that the `GTable` component is not rendered if `!authorizedToViewUsers`, means that technically I can remove this `setState` call 👍

Reply:
this should be addressed in [ee9712f](https://github.com/grafana/oncall/pull/1185/commits/ee9712f92c9264037f077c7d30c327261773907f)